### PR TITLE
Make it clearer that some settings don't have defaults.

### PR DESCRIPTION
**Pull request by bowlofeggs**
Signed-off-by: Randy Barlow <randy@electronsweatshop.com>

### CODE REVIEW

**Comment by bowlofeggs** [comment]
jenkies test

File: production.ini [hunk 1]
```diff
@@ -162,10 +162,11 @@ use = egg:bodhi-server
 #     arches listed in {release}_{version}_primary_arches when it is defined, else used for all
 #     arches. You must put two %s's in this setting - the first will be replaced with the release
 #     version and the second will be replaced with the architecture.
-# fedora_stable_master_repomd = http://download01.phx2.fedoraproject.org/pub/fedora/linux/updates/%s/%s/repodata/repomd.xml
-# fedora_testing_master_repomd = http://download01.phx2.fedoraproject.org/pub/fedora/linux/updates/testing/%s/%s/repodata/repomd.xml
-# fedora_epel_stable_master_repomd = http://download01.phx2.fedoraproject.org/pub/epel/%s/%s/repodata/repomd.xml
-# fedora_epel_testing_master_repomd = http://download01.phx2.fedoraproject.org/pub/epel/testing/%s/%s/repodata/repomd.xml
+# examples (these are not defaults):
```
Comment (pypingou) [inline production.ini]: I think I would be even more obvious: ``these are no the defaults values, there are no default values``

**Review by pypingou — APPROVED** [review]

File: production.ini (hunk 1 above)
Comment (bowlofeggs) [reply production.ini]: Cool, I'll do this instead:

```
diff --git a/production.ini b/production.ini
index 3c9d20b7..a45261bd 100644
--- a/production.ini
+++ b/production.ini
@@ -162,7 +162,7 @@ use = egg:bodhi-server
 #     arches listed in {release}_{version}_primary_arches when it is defined, else used for all
 #     arches. You must put two %s's in this setting - the first will be replaced with the release
 #     version and the second will be replaced with the architecture.
-# examples (these are not defaults):
+# examples (these settings do not have defaults):
 #     fedora_stable_master_repomd = http://download01.phx2.fedoraproject.org/pub/fedora/linux/updates/%s/%s/repodata/repomd.xml
 #     fedora_testing_master_repomd = http://download01.phx2.fedoraproject.org/pub/fedora/linux/updates/testing/%s/%s/repodata/repomd.xml
 #     fedora_epel_stable_master_repomd = http://download01.phx2.fedoraproject.org/pub/epel/%s/%s/repodata/repomd.xml
@@ -176,7 +176,7 @@ use = egg:bodhi-server
 #     arches not listed in {release}_{version}_primary_arches if it is defined. You must put two
 #     %s's in this setting - the first will be replaced with the release version and the second will
 #     be replaced with the architecture.
-# examples (these are not defaults):
+# examples (these settings do not have defaults):
 #     fedora_stable_alt_master_repomd = http://download01.phx2.fedoraproject.org/pub/fedora-secondary/updates/%s/%s/repodata/repomd.xml
 #     fedora_testing_alt_master_repomd = http://download01.phx2.fedoraproject.org/pub/fedora-secondary/updates/testing/%s/%s/repodata/repomd.xml
```

**Comment by bowlofeggs** [comment]
jenkies test

**Pull request merged**